release/ray_release/config.py

```diff
@@ -81,11 +81,20 @@ def _test_definition_invariant(
 
 
 def parse_test_definition(test_definitions: List[TestDefinition]) -> List[Test]:
+    default_definition = {}
     tests = []
     for test_definition in test_definitions:
+        if test_definition["name"] == "DEFAULTS":
+            default_definition = test_definition
```

Comment:
make a deepcopy here? just to be less error prone (and the copy should be cheap)